embedded.go

```diff
@@ -25,9 +32,9 @@ func initHTTPEmbedded(r *mux.Router) {
 }
 
 func handleCreateEmbedded(creq *apps.CallRequest) apps.CallResponse {
-	client := appclient.AsBot(creq.Context)
+	client := appclient.AsActingUser(creq.Context)
 	p := &model.Post{
-		ChannelId: creq.Context.ChannelID,
+		ChannelId: creq.Context.Channel.Id,
```

Comment:
Is it possible for us to "hide" the `Context.ChannelID` field that will never be defined?

Reply:
At some point, yes. But it requires a restructuring: https://github.com/mattermost/mattermost-plugin-apps/blob/4d52c70c63c1e77e83ef7241fe15ff2e3954f54b/apps/context.go#L18